### ISSUE TITLE
Can't Use Any Model Except fcn_8

### ISSUE THREAD

**Issue by abertonica** [divamgupta/image-segmentation-keras#336]
Any time I try to use any model except fcn_8 I get this error:

weights_path = keras.utils.get_file(
AttributeError: module 'keras.utils' has no attribute 'get_file'

Any ideas?

Thanks

**Comment by sushantag9** [comment]
Everywhere update

`import keras` to `import tensorflow.keras`

This will solve the issue. 

You can also clone this repo and use it [image-segmentation-keras](https://github.com/sushantag9/image-segmentation-keras)

**Comment by santra11** [comment]
Hello Sushantag9, I cloned your repository. I have my current work directory, I cloned your Keras segmentation and I have a Jupyter notebook. Now I use the `model = vgg_unet(n_classes=n_classes,input_height=img_height,input_width=img_width)` gives the error keras.utils has no attribute.

**Comment by sushantag9** [comment]
Can you share the full error.

**Comment by santra11** [comment]
No, I sorted it out. Just a question. I am working with cityscapes dataset and using custom_augmentation exactly used in Divyam Gupta, but getting an error for the mobile net segnet mode
![Screenshot 2021-12-29 at 18 20 22](https://user-images.githubusercontent.com/75618330/147687692-dd6d9107-b61f-41e6-b3d3-b27af60d172e.png)
and 
![Screenshot 2021-12-29 at 18 15 44](https://user-images.githubusercontent.com/75618330/147687792-00d09fa4-1cfa-4d81-9af3-4ebec659b586.png)

Can anyone help. If I do without augmentation, averaging works fine.

**Comment by subramanyakrishna** [comment]
Please install from  @sushantag9  's  repo
replace 
from tensorflow.keras.layers.merge import Concatenate
with 
from tensorflow.keras.layers import Concatenate
it worked for me

**Comment by santra11** [comment]
Hello, Sushant,
   I am using the image segmentation for cityscapes pertained models. When I do model.train() , what is mean IoU  considered as the metric. I am not able to to understand  as it is different from normal the mean IoU. Also how to plot the history of the models.

**Comment by omarreess** [comment]
Just import those 

`!apt-get install -y libsm6 libxext6 libxrender-dev
!pip install opencv-python
!pip install git+https://github.com/divamgupta/image-segmentation-keras
from keras_segmentation.models.unet import vgg_unet`
